docs/source/guide/calibrators.md

```diff
@@ -32,17 +32,9 @@ from mitiq import MeasurementResult
 ```{code-cell} ipython3
 def execute(circuit, noise_level=0.001):
     circuit = circuit.with_noise(cirq.amplitude_damp(noise_level))
-
     result = cirq.DensityMatrixSimulator().run(circuit, repetitions=100)
-    return MeasurementResult(
-        result=np.column_stack(list(result.measurements.values())).tolist(),
-        qubit_indices=tuple(
-            # q[2:-1] is necessary to convert "q(number)" into "number"
-            int(q[2:-1])
-            for k in result.measurements.keys()
-            for q in k.split(",")
-        ),
-    )
```

Comment:
Why are the lines specifying the qubit indices removed? How do we keep track of the qubit indices associated with the bitstring if they aren't part of the `MeasurementResult`?

Reply:
For the calibrator it is probably ok to use the default value for `qubit_indices` since all qubits must be measured during the calibration process.